packages/atlas/atlas.config.yml

```diff
@@ -84,6 +84,12 @@ features:
         baseAmount: 250
         actionButtonText: Get referral link
         actionButtonAction: copyReferral
+    widgets: # Widgets on Ypp landing page
+      - title: Discord
+        link: https://discord.gg/DE9UN3YpRP
+        linkText: Go to Discord # Used only on YPP Dashboard - if empty defaults to "Go to {title}"
+        vendor: Discord # Used for YPP Dashboard to inform user which vendor given feature uses - if empty defaults to title
```

Comment:
I'm not sure if I understand what's a vendor. Is it a label? This thing? 👇🏻 
![image](https://user-images.githubusercontent.com/51168865/208065764-4123920b-3dad-40dc-8747-906706459cbe.png)
 If yes, can we just rename this to label?